projects/storefrontapp-e2e-cypress/cypress/integration/regression/cart/clear-cart.e2e-spec.ts

```diff
@@ -0,0 +1,41 @@
+import * as cart from '../../../helpers/cart';
+import { visitHomePage } from '../../../helpers/checkout-flow';
+import { viewportContext } from '../../../helpers/viewport-context';
+
+let cartId;
```

Comment:
Perhaps we add a type to this var.

Reply:
@plabadie Well this var is no longer needed for the tests so I removed it. It should be removed from PR  in next push with the rest of the test cases